setup.py

```diff
@@ -212,7 +212,8 @@ def main(**extra_args):
                           'dipy.sims',
                           'dipy.sims.tests',
                           'dipy.denoise',
-                          'dipy.denoise.tests'],
+                          'dipy.denoise.tests',
+                          'dipy.workflows'],
```

Comment:
I think you want to add `dipy.workflows.tests` in here, so that the tests also get installed